### PR TITLE
add inline scrims

### PR DESCRIPTION
**Pull request by LeoMcA**
### Description

Add inline scrims throughout curriculum.

Needs to wait for the related yari PR to be merged before merging.

### Related issues and pull requests

- https://mozilla-hub.atlassian.net/browse/MP-1508
- https://github.com/mdn/yari/pull/11866

@chrisdavidmills could you check these scrims correspond to the ones you put in the doc? Feel free to update any wording/move things around as you see fit. I'll get this up on stage so you can see how it looks.

### CODE REVIEW

File: curriculum/3-core/5-css-layout.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ Resources:
 
 - [Stacking context](https://developer.mozilla.org/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)
 
+<scrim-inline url="https://v2.scrimba.com/the-frontend-developer-career-path-c0j/~0d5" scrimtitle="Relative and Absolute Position"></scrim-inline>
```
Comment (chrisdavidmills) [inline curriculum/3-core/5-css-layout.md:77]: On reflection, I think this one feels a bit out of place (I appreciate you've put it where I asked you to!)

Instead, can you put it below the "Setting the positioning context of a positioned element by positioning an ancestor element." line?

**Review by chrisdavidmills — APPROVED** [review]
Great stuff @LeoMcA. I've checked it against my Google Doc, and all of the URLs, display titles and embed positions look to be correct. I just had a couple of minor comments.

File: curriculum/3-core/1-web-standards.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Learning outcomes:
 
 - Clients and servers and their roles in the web.
 
+<scrim-inline url="https://v2.scrimba.com/the-frontend-developer-career-path-c0j/~0lr" scrimtitle="The Request-Response Cycle"></scrim-inline>
```
[the remark this answers is not in the document]
Comment (LeoMcA) [reply curriculum/3-core/1-web-standards.md:21]: It depends how we want the markup reflected, really. Un-indented (like it is currently) results in a markup like:
```html
  <li>List element before</li>
</ul>
<p><scrim-inline /></p>
<ul>
  <li>List element after</li>
```

Whereas intending it results in:
```html
  <li>
    List element before
    <scrim-inline />
  </li>
  <li>List element after</li>
```

[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply curriculum/3-core/1-web-standards.md:21]: I think the second one would be better. The first one results in a bunch of chopped-up, broken lists that visually look OK, but would be trickier to navigate for AT users. 

Let's maybe not worry about fixing this now, but look at it in a separate PR?

**Comment by chrisdavidmills** [comment]
Perfect, I think this is ready to go. I will hold on merging it until the associated yari work is ready.

**Comment by LeoMcA** [comment]
@chrisdavidmills The scrims are on staging if you'd like to take a look: https://developer.allizom.org/en-US/curriculum/getting-started/soft-skills/#1.4_collaboration_and_teamwork

**Comment by chrisdavidmills** [comment]
@LeoMcA OK, I've had a look and I think these look nice and function well. My only outstanding comments are:

- When you click to load the scrim and it pops up full window width, you get a little message popping up at the bottom of the embed that says "OPError: Not Allowed". The embed works fine, but I wonder if this might put some users off a bit. Not sure if this is something that needs to be looked at on our side of Scrimba's side?
- The embeds in the page are the full width of the content column. IMO, they look a bit big and in-your-face on desktop. Would it be worth considering making them a bit smaller (say 70% of column width) on desktop, and centering them vertically?
- In my Google doc, Miruna expressed concern that one of the scrims was also linked to as a regular link. I thought about this, and wondered if we ought to remove the links that link to the same embedded scrims. Does it seem weird, or is it helpful to have regular link versions available too? Or maybe we should just remove the links in cases where you've got the embed right next to the regular link. What do you think?

File: curriculum/3-core/1-web-standards.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LeoMcA) [reply curriculum/3-core/1-web-standards.md]: I needed to do a redeploy anyway to make the scrims a little smaller, so decided to do this now in https://github.com/mdn/curriculum/pull/73/commits/406270f57841114cf2258132c5fdc44bea03a4e2

**Comment by LeoMcA** [comment]
> When you click to load the scrim and it pops up full window width, you get a little message popping up at the bottom of the embed that says "OPError: Not Allowed". The embed works fine, but I wonder if this might put some users off a bit. Not sure if this is something that needs to be looked at on our side of Scrimba's side?

Yes, I need to raise this with Scrimba

> The embeds in the page are the full width of the content column. IMO, they look a bit big and in-your-face on desktop. Would it be worth considering making them a bit smaller (say 70% of column width) on desktop, and centering them vertically?

I've made them a little smaller, will deploy the change to staging

> In my Google doc, Miruna expressed concern that one of the scrims was also linked to as a regular link. I thought about this, and wondered if we ought to remove the links that link to the same embedded scrims. Does it seem weird, or is it helpful to have regular link versions available too? Or maybe we should just remove the links in cases where you've got the embed right next to the regular link. What do you think?

I'd keep the regular links, at least for now: we haven't fully figured out server side rendering of custom elements yet, and I don't want that to be a release blocker. Without it, if the custom element doesn't load (due to client side JS being disabled, or other reasons), and if we deduplicate the links, we'd end up with no link to the scrim. The PDF also requires the links, as it just ignores the custom element.

**Comment by chrisdavidmills** [comment]
> I've made them a little smaller, will deploy the change to staging

@LeoMcA The new size looks great; GTG from my end.

**Pull request merged**